media-libs/libltc/libltc-9999.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Linear/Logitudinal Time Code (LTC) Library"
+HOMEPAGE="https://github.com/x42/libltc.git"
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3 autotools
+	EGIT_REPO_URI="https://github.com/x42/libltc.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/x42/libltc/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+LICENSE="LGPL-3"
+SLOT="0"
+RESTRICT="mirror"
```

Comment:
Why this restriction? I thought this was only for licensing/bindist issues.

Reply:
I think this is something that all overlays should be using because otherwise portage first tries to download a copy of the SRC_URI(s) from the gentoo mirrors, which obviously doesn't work for pretty much all our packages.
(This was already mentioned on IRC, figured I should post it here as well).

Reply:
Yeah, sounds reasonable :)